### PR TITLE
Exporting ds_reload to cfg script

### PR DESCRIPTION
**Pull request by luismartingil**

### CODE REVIEW

File: modules/dispatcher/doc/dispatcher_admin.xml [hunk 1]
```diff
@@ -1350,6 +1350,20 @@ onreply_route {
 </programlisting>
 		</example>
  	</section>
+	<section id="dispatcher.f.reload">
```
Comment (oej) [inline modules/dispatcher/doc/dispatcher_admin.xml]: dispatcher.f.ds_reload is correct

File: modules/dispatcher/doc/dispatcher_admin.xml [hunk 2]
```diff
@@ -1350,6 +1350,20 @@ onreply_route {
...
+		<title>
+		<function moreinfo="none">ds_reload()</function>
+		</title>
+		<para>
+		Reloads the groups and included destinations. The command is
+		disabled for call load based dispatching (algorithm 10) since
```
Comment (oej) [inline modules/dispatcher/doc/dispatcher_admin.xml]: The code doesn't indicate that it really is disabled in this situation. Do you mean "disabled" or "should not be used".

**Comment by oej** [comment]
You can already run rpc commands from the script. What does this patch add?

**Comment by luismartingil** [comment]
Thanks for the input. How would you run the RPC commands from the script?

File: modules/dispatcher/doc/dispatcher_admin.xml (hunk 2 above)
Comment (luismartingil) [inline modules/dispatcher/doc/dispatcher_admin.xml]: This is based on [this](https://github.com/kamailio/kamailio/blob/master/modules/dispatcher/dispatcher.c#L914-L924) and [this](https://github.com/kamailio/kamailio/blob/master/modules/dispatcher/doc/dispatcher_admin.xml#L1415-L1437). I copied the possible problems description using a similar function.

**Comment by oej** [comment]
There is support in the jsonrpc-s module. Kind of hidden, but still there :-)

http://kamailio.org/docs/modules/4.4.x/modules/jsonrpc-s.html#jsonrpc-s.f.jsonrpc_exec

**Comment by luismartingil** [comment]
Didn't see that one coming! ;-)

**Comment by miconda** [comment]
It's ok to export this function directly, can be handy sometimes to use it directly, without rpc workaround.

But the description needs to be amended, because the reload is not disabled anymore for algorthm 10 (it was some time ago).

**Comment by luismartingil** [comment]
Even though I see you're solution as a valid solution, I still see this exported function as valuable. Using RPC to trigger an internal mechanism can overload this procedure.

**Comment by luismartingil** [comment]
@miconda I'll change the description.

**Comment by oej** [comment]
I agree that it is useful as a simplification, but wanted to know if there was some additional benefits. You may want to add counters on how many dispatcher targets and groups that was loaded or something like that while you are at it - or a checksum check so I can check if the list was actually changed. Just some ideas to make this function an improvement :-)

**Comment by luismartingil** [comment]
@miconda I'll just leave this _PR_ as it is, please let me know if you want to integrate this change and if you'll merge the changes and close the _PR_ or you want me to do it. Thanks!

**Pull request merged**